dh_collections_prism_collection_community.json

```
@@ -68,18 +68,26 @@
     "collection_id": "A genome-wide association study (GWAS) of bitter and sweet beverage consumption",
     "community_id": "researchers-collections"
   },
-  "d0798568-47c3-453e-ae39-242d8a96b1dc": {
-    "collection_id": "Aberrant cGMP Signaling Persists During Recovery in Mice with Oxygen-Induced Pulmonary Hypertension",
-    "community_id": "researchers-collections"
-  },
   "913f8fa2-06c9-49e1-9cdf-0f88118b18da": {
     "collection_id": "A Polygenic and Phenotypic Risk Prediction for Polycystic Ovary Syndrome evaluated by Phenome-wide association studies - Supplemental Material",
     "community_id": "researchers-collections"
   },
+  "2dd287f0-9748-41f7-8fab-222db450d196":{
+    "collection_id": "A retrospective look at the predictions and recommendations from the 2009 AMIA Policy Meeting",
+    "community_id": "researchers-collections"
+  },
+  "d0798568-47c3-453e-ae39-242d8a96b1dc": {
+    "collection_id": "Aberrant cGMP Signaling Persists During Recovery in Mice with Oxygen-Induced Pulmonary Hypertension",
+    "community_id": "researchers-collections"
+  },
   "8s45q876k": {
     "collection_id": "Beyond an Apple a Day: Finding Reliable Health Information Online",
     "community_id": "researchers-collections"
   },
+  "5f1b1739-512f-4015-98bc-22f37f42af7b": {
+    "collection_id": "calantone-et-al-2022-commun-biol-source-data",
```

Double-checking here, because this looks slightly different than the rest. At a glance, all other `collection_id` don't have this format.

---

It looks like this entry was just moved up the list from line 422

---

I think what @sharpattack means is that for other entries the `collection_id` is not in a slug format but rather a human readable format. So here it would be: 

 ```suggestion
    "collection_id": "Calantone et al 2022 Commun Biol Source Data",
```

Since we are not using collection_ids right now it's not a big deal, but might as well change it.